src/runtime/pkg/oci/utils.go

```diff
@@ -576,6 +574,37 @@ func addHypervisorPathOverrides(ocispec specs.Spec, config *vc.SandboxConfig, ru
 	return nil
 }
 
+func addHypervisorPCIePortOverride(value string) (config.PCIePort, error) {
+	if value == "" {
+		return config.NoPort, nil
+	}
+	port := config.PCIePort(value)
+	if port.InValid() {
```

Comment:
Weird capitalization fix
```suggestion
	if port.Invalid() {
```